.github/workflows/ci-cd.yml

```
@@ -38,11 +38,14 @@ jobs:
         - 3.11
         - 3.12
         - 3.13
+        - 3.14
```

If you use
```suggestion
        - ~3.14.0-0
```
or
```suggestion
        - 3.14-dev
```
then you could check for “starts with a tilde or ends with a dash dev” in the reusable workflow and have the experimental flag built-in. For any additional experimental states (OS/arch-dependent), you could use `experimental` explicitly (and stop using `allow-prereleases` in there too).